app/controllers/concerns/blacklight/default_component_configuration.rb

```diff
@@ -64,39 +65,26 @@ def add_show_tools_partial name, opts = {}
         end
       end
 
-      ##
       # Add a tool to be displayed for each document in the search results.
-      # @param partial [String] the name of the document partial
-      # @param opts [Hash]
-      # @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true.
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      # @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      def add_results_document_tool name, opts = {}
+      # @see #add_nav_action for params
```

Comment:
I don't think the `see` looks great actually, but I _still_ think ti's better than copy-and-pasted duplicated tests. I think copy-and-pasted duplicated tests makes it even more likely that at least one place won't be kept up to date when the parameters change. Blacklight has not always done so well at keeping documented method signatures up to date even without duplication. I'd rather have docs with slightly less good UX if they are more likely to be up to date; a great UX on out of date docs helps nobody.